buildpack.md

```diff
@@ -1,8 +1,8 @@
 # Buildpack Interface Specification
 
-This document specifies the interface between a single lifecycle and one or more buildpacks.
+This document specifies the interface between a program called `lifecycle` and one or more buildpacks.
```

Comment:
A lifecycle is a type of program, not a specific program. The lifecycle provided by the project is called `lifecycle`, but not all lifecycles must use that name.

Reply:
Type of program is an interface. The definition will then reads as something like "documents specifies interfaces for interface programs to handle buildpacks". Maybe just say name that "Buildpack Reference" and the then explain what a buildpack is, what `lifecycle` program does, and what is the common interface for handling buildpacks (with or without lifecycle).

Reply:
Hi @abitrolly, I wonder if a more appropriate place to provide these sort of definitions is on our [docs page](https://buildpacks.io/docs/concepts/components/lifecycle/). The spec is more focused on declaring concepts in an abstract way that various implementations can satisfy. The docs site can more concretely speak to and reference implementations that currently exist.

I'd be happy to work through this via [slack](https://slack.buildpacks.io/) (`#learning-team`) or [mailing list](https://lists.cncf.io/g/cncf-buildpacks).

Reply:
@jromero both places can serve as an entrypoint, so both should be fixed. A person who would read these abstract terms would likely go and search for another explanation, which devalues time that various people put into writing these documents.

Reply:
I completely agree. I just wanted to make sure we keep in mind that the context would be different. There are users of buildpacks and the implementations of various buildpack components of the spec. To me it seemed like the initial proposal was gearing towards users of buildpacks. The latest update clarifies it for me.

Reply:
Made one more try.